README.md

```diff
@@ -1,5 +1,25 @@
 WebGL 2 Samples Pack
 =====================
 
-Samples demonstrating WebGL 2 features by [Shuai Shao (Shrek)](https://www.linkedin.com/in/shuai-shao-3718818b) and [Trung Le](http://www.trungtuanle.com/). Advised by [Patrick Cozzi](http://www.seas.upenn.edu/~pcozzi/), University of Pennsylvania. 
+GH-Page: http://webglsamples.org/WebGL2Samples/
 
+Samples demonstrating WebGL 2 features by [Shuai Shao (Shrek)](https://shrekshao.github.io) and [Trung Le](http://www.trungtuanle.com/). Advised by [Patrick Cozzi](http://www.seas.upenn.edu/~pcozzi/), University of Pennsylvania.
+
+## Samples list
+
+|name          |  port from | Chrome 47 Win | Firefox 42 Win|
+|--------------|------------|:---------------:|:---------------:|
+|draw instanced|  [gl-320-draw-instanced](https://github.com/g-truc/ogl-samples/blob/master/tests/gl-320-draw-instanced.cpp)|   :o:    |  :x: crash        |
```

Comment:
Does the red "o" mean it passes?  No one will get that.  Red generally means something bad.  Try :white_check_mark:.